sdk/resourcemanager/azure-resourcemanager-storage/CHANGELOG.md

```diff
@@ -4,9 +4,7 @@
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- Supported toggling blob versioning in `BlobServiceProperties`.
```

Comment:
Delete other entries (e..g breaking changes etc.).

Or remember to manually modify the changelog in the PR preparing release (the script won't erase them).

Else release pipeline will fail on it (or maybe CI can catch it before release).

Reply:
OK, thanks!